utils/check_repo.py

```diff
@@ -37,6 +37,7 @@
     "TFDPRSpanPredictor",  # Building part of bigger (tested) model.
     "TFElectraMainLayer",  # Building part of bigger (tested) model (should it be a TFPreTrainedModel ?)
     "TFRobertaForMultipleChoice",  # TODO: fix
+    "TFMPNetForMultipleChoice",  # TODO: fix
```

Comment:
Do we have someone working on this?

Reply:
@sgugger I add one line to address this problem. But I am not sure its style is ok in the huggingface implementation.

https://github.com/StillKeepTry/transformers/blob/78dcc71fd96ec6a04739a422f38bab0532e42a8d/src/transformers/models/mpnet/modeling_tf_mpnet.py#L175

This problem seems is because the position ids from fairseq are different from others (which is started from 2). Since our model is also implemented in fairseq, which will report such error (same as roberta).

Reply:
@sgugger Besides, do you have any other comments.